unison-src/transcripts/docs.output.md

```diff
@@ -158,13 +158,11 @@ Now that documentation is linked to the definition. We can view it if we like:
   ## Examples:
   
   List.take.ex1 = builtin.List.take 0 [1, 2, 3, 4, 5]
-  🔽
-  List.take.ex1 = []
+  🔽 ex1 = []
   
   
   List.take.ex2 = builtin.List.take 2 [1, 2, 3, 4, 5]
-  🔽
-  List.take.ex2 = [1, 2]
+  🔽 ex2 = [1, 2]
```

Comment:
@runarorama weird what happened to the line breaks here? Is that related to your change? Also unfortunate that the link expands to the suffixed name, not the fully qualified name, I think it makes this idiom of `@blah = @[evaluate] blah` less useful. Maybe just:

```
@[source] blah
⧩
@[evaluate] blah
```

Is fine, but it should preserve those line breaks.

Reply:
It has always been the case that

```
doc = [:
  foo
  bar 
  baz
:]
```

displays as

```
foo bar baz
```

Is that incorrect?

Reply:
@runarorama can you explain why the output has changed? Previously it used to preserve the line break after the 🔽.

I'm not sure it's correct... I think it depends what we want. Current behavior is like markdown where single line breaks are turned into spaces. We'd need an explicit way to insert single line breaks if we keep this behavior.

@atacratic Would it work to just preserve all line breaks? The main thing we wanted to be able to do is dynamically rewrap the document on display to fit the terminal width.

Reply:
It didn't preserve a line break after the 🔽, but inserted one before the `evaluate` output.

Reply:
K. Let's merge this then. Can rethink the doc display / rewrapping separately.